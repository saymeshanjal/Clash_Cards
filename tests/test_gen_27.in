7
16 163 2
10 36 1
10 36 4
14 300 2
16 52 1
6 289 4
8 387 4
