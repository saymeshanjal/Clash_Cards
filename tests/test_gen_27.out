4081 4082 4707
126 126 0
43 133 76
1078 1079 1275
6359 6511 0
1 14 5
8 42 20
